gammapy/image/mask.py

```diff
@@ -2,217 +2,27 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 import numpy as np
 from astropy.coordinates import Latitude, Longitude, Angle
-from astropy.utils import lazyproperty
+from ..image.core import SkyImage
 from ..image import lon_lat_circle_mask
-from .core import SkyImage
 
 __all__ = [
-    'SkyMask',
     'make_tevcat_exclusion_mask'
 ]
 
 
-class SkyMask(SkyImage):
-    """Sky image mask.
-
-    `SkyMask` is a `~gammapy.image.SkyImage` sub-class, i.e. it inherits
-    all of its features. The distinction is that `SkyMask` is to
-    represent boolean masks and has methods that only make sense for
-    mask data. The data array can be integer or float, but if it is,
-    it should only contain pixel values of 0 or 1.
-
-    TODO: explain about semantics and give examples what 0 and 1 mean
-    in different applications (or link to other docs).
-    """
-
-    def fill_random_circles(self, n=4, min_rad=0, max_rad=40):
```

Comment:
I don't think it's a good idea to bring this back. It's just something random and complicated, where in tests or examples something non-random and simpler (like a circle or box) would be better.

If you must, please put it somewhere where it's not visible to users (e.g. if it's in one of the test py files you can still import it from everywhere).